asn1acn.pro

```diff
@@ -128,6 +128,8 @@ SOURCES += \
     libraries/modulemetadataparser.cpp \
     libraries/generalmetadataparser.cpp \
     \
+    component-importer\importcomponentdialog.cpp \
```

Comment:
I wonder should it have it's own folder or be part of `libraries` folder (or have some subdir there)

Reply:
Added it to `libraries` , removed subfolder. I did not know what the `libraries` for at first, but I think new files fit there.